### PR TITLE
Issue/29 make data mutable

### PR DESCRIPTION
**Pull request by smholloway**
This PR simulates the Flask `request.args` being sent to `ToopherIframe.validate()`.

### CODE REVIEW

**Comment by smholloway** [comment]
See Issue #29

File: requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ nose==1.3.0
 oauthlib==0.6.0
 requests==2.0.1
 requests-oauthlib==0.4.0
+Werkzeug==0.9.4
```
Comment (dshafer) [inline requirements.txt:6]: Any way to make this a test-only dependency?

Comment (trdarr) [inline requirements.txt:6]: The dependencies in `requirements.txt` are for developing the library. Installation dependencies are defined in `setup.py`.

**Comment by dshafer** [comment]
Github really needs to steal bitbucket's "I Approve this Pull Request" button.

Also, I approve this pull request.

**Comment by smholloway** [comment]
BTW when `data` comes in as a `werkzeug.datastructures.ImmutableMultiDict`, `data = dict(data)`  returns a dictionary with single-item arrays, which is why I put it above the `flatten` logic. It's subtle so I thought it might be worth mentioning.

**Comment by smholloway** [comment]
Will merge tomorrow afternoon if no one objects.

**Pull request merged**